src/main/java/io/elastic/sailor/AmqpAwareModule.java

```diff
@@ -9,12 +10,12 @@
 public class AmqpAwareModule extends AbstractModule {
     @Override
     protected void configure() {
-        bind(AmqpService.class).to(AmqpServiceImpl.class);
-        bind(MessageProcessor.class).to(MessageProcessorImpl.class);
-        bind(MessagePublisher.class).to(MessagePublisherImpl.class);
-        bind(ErrorPublisher.class).to(ErrorPublisherImpl.class);
-        bind(MessageResolver.class).to(MessageResolverImpl.class);
-        bind(ObjectStorage.class).to(ObjectStorageImpl.class);
+        bind(AmqpService.class).to(AmqpServiceImpl.class).in(Scopes.SINGLETON);
```

Comment:
AmqpService is already a singleton. Please see here https://github.com/elasticio/sailor-jvm/blob/master/src/main/java/io/elastic/sailor/impl/AmqpServiceImpl.java#L17

Reply:
Ok. I will use annotation style for every place where it should be.